src/commands/scanner/rule/add.ts

```diff
@@ -17,32 +17,34 @@ const messages = Messages.loadMessages('@salesforce/sfdx-scanner', 'add');
 
 export default class Add extends ScannerCommand {
 
-	public static description = messages.getMessage('commandDescription');
-	public static longDescription = messages.getMessage('commandDescriptionLong');
+	public static summary = messages.getMessage('commandDescription');
+	public static description = messages.getMessage('commandDescriptionLong');
 
 	public static examples = [
 		messages.getMessage('examples')
 	];
 
-	protected static flagsConfig = {
```

Comment:
According to the CLI team, it seems like it's because we were using `flagsConfig` instead of `flags`, and because we had abstract classes between `SfdxCommand` and our actual commands.